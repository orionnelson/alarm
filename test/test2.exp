alarm> Main Thread Received Alarm Request Number: (1) Alarm Request: <hello> 
alarm> Main Thread Received Alarm Request Number: (2) Alarm Request: <hi> 
alarm> Main Thread Received Alarm Request Number: (3) Alarm Request: <hey> 
alarm> Alarm Thread Passed on Alarm Request to Display Thread 2 Alarm Request Number: (2) Alarm Request: hi 
Display Thread 2: Recieved Alarm Request Number:(2) Alarm Request: <hi> 
Display Thread 2: Number of SecondsLeft <9>: Alarm Request Number: (2) Alarm Request: <hi> 
Display Thread 2: Number of SecondsLeft <7>: Alarm Request Number: (2) Alarm Request: <hi> 
Display Thread 2: Number of SecondsLeft <5>: Alarm Request Number: (2) Alarm Request: <hi> 
Display Thread 2: Number of SecondsLeft <3>: Alarm Request Number: (2) Alarm Request: <hi> 
Display Thread 2: Number of SecondsLeft <1>: Alarm Request Number: (2) Alarm Request: <hi> 
Display Thread 2: Alarm Expired at 1637030221: Alarm Request Number: (2) Alarm Request hi 
Alarm Thread Passed on Alarm Request to Display Thread 1 Alarm Request Number: (1) Alarm Request: hello 
 Display Thread 1: Recieved Alarm Request Number:(1) Alarm Request: <hello> 
Display Thread 1: Number of SecondsLeft <9>: Alarm Request Number: (1) Alarm Request: <hello> 
Display Thread 1: Number of SecondsLeft <7>: Alarm Request Number: (1) Alarm Request: <hello> 
Display Thread 1: Number of SecondsLeft <5>: Alarm Request Number: (1) Alarm Request: <hello> 
Display Thread 1: Number of SecondsLeft <3>: Alarm Request Number: (1) Alarm Request: <hello> 
Display Thread 1: Number of SecondsLeft <1>: Alarm Request Number: (1) Alarm Request: <hello> 
Display Thread 1: Alarm Expired at 1637030231: Alarm Request Number: (1) Alarm Request hello 
Alarm Thread Passed on Alarm Request to Display Thread 1 Alarm Request Number: (3) Alarm Request: hey 
 Display Thread 1: Recieved Alarm Request Number:(3) Alarm Request: <hey> 
Display Thread 1: Number of SecondsLeft <9>: Alarm Request Number: (3) Alarm Request: <hey> 
Display Thread 1: Number of SecondsLeft <7>: Alarm Request Number: (3) Alarm Request: <hey> 
Display Thread 1: Number of SecondsLeft <5>: Alarm Request Number: (3) Alarm Request: <hey> 
Display Thread 1: Number of SecondsLeft <3>: Alarm Request Number: (3) Alarm Request: <hey> 
Display Thread 1: Number of SecondsLeft <1>: Alarm Request Number: (3) Alarm Request: <hey> 
Display Thread 1: Alarm Expired at 1637030241: Alarm Request Number: (3) Alarm Request hey 
