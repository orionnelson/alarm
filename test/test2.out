Bad command
Bad command
Bad command
alarm> alarm> alarm> alarm> 